src/app/frontend/replicasetdetail/deletereplicaset_service.js

```diff
@@ -0,0 +1,67 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {StateParams} from './replicasetdetail_state';
+import {getReplicaSetDetailsResource} from './replicasetdetail_stateconfig';
+
+/**
+ * Opens replica set delete dialog.
+ *
+ * @final
+ */
+export class DeleteReplicaSetService {
```

Comment:
I think that for now it can be for delete only. Later, when somebody wants to add new functionality to this service, the name can be changed. I think this is a better approach, because with it we do not design for the (unknown) future :) Only for what is now.